### DISCUSSION TITLE
Small tile icon size is too big compared to other apps

### DISCUSSION THREAD

**Discussion by SOI7** [character-map-uwp/Character-Map-UWP#198] [Ideas]
![image](https://user-images.githubusercontent.com/50930970/106385540-16fe2080-63d1-11eb-9a99-43966a9da8b3.png)

As you can see in the screenshot, the app icon is too exaggerated compared to other app icons when using Small Tiles. I think it should be reduced to be in line with the other apps

**Comment by ClewenWei** [comment]
The icon in a wide tile is too wide itself as it could to display, too.
![图片](https://user-images.githubusercontent.com/57256934/111481055-8c4d6700-876d-11eb-98ca-6ac63c9fdd9e.png)
